modules/gapi/src/compiler/gislandmodel.hpp

```diff
@@ -123,6 +123,7 @@ class GAPI_EXPORTS GIslandExecutable
     virtual bool canReshape() const = 0;
     virtual void reshape(ade::Graph& g, const GCompileArgs& args) = 0;
 
+    virtual void prepareForNewStream() { }; // do nothing here by default
```

Comment:
😿 
The comment below explains the thing.

Reply:
This method is removed from GIslandExecutable.